app/index.html

```diff
@@ -15,37 +15,19 @@
     {{content-for 'head-footer'}}
   </head>
   <body>
-    <div class="jumbotron" id="apploading">
-        <div class="alert alert-warning" role="alert">
-            <h1><span class="glyphicon glyphicon-refresh glyphicon-refresh-animate"></span>HospitalRun is Initializing... </h1>
-            <p>Did You Know: <span id="apploadingmsg"></span></p>
+    <div class="loading-notice apploading" id="apploading" role="alert">
+        <div class="wrapper">
+            <div class="spinner">
+                <div class="rect1"></div>
+                <div class="rect2"></div>
+                <div class="rect3"></div>
+                <div class="rect4"></div>
+                <div class="rect5"></div>
+            </div>
+            <p class="loading-text">Loading</p>
         </div>
     </div>
     <script>
-        var messages = [
-            'The Brimstone butterfly (Gonepteryx rhamni) has the longest lifetime of the adult butterflies: 9-10 months.',
-            'Apples are more efficient than caffeine in keeping people awake in the mornings.',
-            'The human heart creates enough pressure when it pumps out to the body to squirt blood 30 feet.',
-            'Starfish have no brains.',
-            'A dinomaniac is someone with the compulsive urge to dance.',
-            'There is no leading cause of death for people who live past the age of 100.',
-            'Nepal is the only country that doesn’t have a rectangular flag.',
-            'The international telephone dialing code for Antarctica is 672.',
-            'Camels chew in a figure 8 pattern.',
-            'A shark is the only fish that can blink with both eyes.'
-        ]
```

Comment:
>Actually in the Philippines with a large data set, some screens need something like this because some screens take a while to load.

Okay, well if that's the case, I understand why they were there, but still think having something generic (like the loading animation) is more appropriate.